mbuild/tests/test_compound.py

```diff
@@ -117,16 +118,31 @@ def test_save_forcefield_with_file(self, methane):
                          forcefield_files=get_fn('methane_oplssaa.xml'),
                          overwrite=True)
 
-    @pytest.mark.skipif(not has_foyer, reason="Foyer is not installed")
-    def test_save_forcefield_with_file_foyerkwargs(self, methane):
-        foyerkwargs = {'assert_improper_params': True}
+    @pytest.mark.parametrize("ff_filename,foyer_kwargs", [
+        ("ethane-angle-typo.xml", {"assert_angle_params": False}),
+        ("ethane-dihedral-typo.xml", {"assert_dihedral_params": False})
+    ])
+    def test_save_missing_topo_params(self, ff_filename, foyer_kwargs):
+        """Test that the user is notified if not all topology parameters are found."""
+        from foyer.tests.utils import get_fn
+        ethane = mb.load(get_fn('ethane.mol2'))
         with pytest.raises(Exception):
-            methane.save('lythem.hoomdxml',
-                             forcefield_files=get_fn('methane_oplssaa.xml'),
-                             overwrite=True, foyerkwargs=foyerkwargs)
-        methane.save('lythem.hoomdxml',
-                forcefield_files=get_fn('methane_oplssaa.xml'),
-                overwrite=True, foyerkwargs={})
+            ethane.save('ethane.mol2', forcefield_files=get_fn(ff_filename))
+        with pytest.warns(UserWarning):
+            ethane.save('ethane.mol2', forcefield_files=get_fn(ff_filename),
+                        overwrite=True, foyer_kwargs=foyer_kwargs)
+#=======
```

Comment:
Looks like commented out merge conflicts are still here.

Reply:
I think it's probably safe to un-comment and just refactor `foyerkwargs` -> `foyer_kwargs` in this test?

Reply:
You were right, it was just a mistake from me resolving merge conflicts